libs/api/domains/education/src/lib/graphql/inna.resolver.ts

```diff
@@ -27,7 +27,7 @@ import { DiplomaModel } from './inna/diplomas.model'
 @FeatureFlag(Features.servicePortalSecondaryEducationPages)
 @Audit({ namespace: '@island.is/api/education-inna' })
 export class InnaResolver {
-  constructor(private innaService: InnaService) {}
+  constructor(private innaService: InnaClientService) {}
```

Comment:
Seeing some warnings from Github about 
`Type 'InlineResponse2001 | null' is not assignable to type 'PeriodsModel | null'.`

Does this change trigger this or is it something incidental that can be ignored?

Reply:
Yea I need to fix it before merging, missing null value in model